tests/PHPUnit/Unit/Config/test_files/boolean_settings_test.ini.php

```diff
@@ -0,0 +1,8 @@
+[BoolSettings]
+one = 1
+onestr = '1'
+two = 2
+twostr = '2'
+invalid = 'invalid'
+truebool = true
```

Comment:
you could also add a check for the values `ON` and `YES`, which should be converted to `true` as well as `OFF` and `NO`, which should be converted to `false` by the ini component. 
Also there might be edge cases like `1test`, `te1st`, which may be `1` when converted to int.